d2/src/main/java/com/linkedin/d2/balancer/D2ClientBuilder.java

```diff
@@ -643,6 +644,11 @@ public D2ClientBuilder setDualReadStateManager(DualReadStateManager dualReadStat
     return this;
   }
 
+  public D2ClientBuilder setLoadBalancerExecutor(ExecutorService loadBalancerExecutor) {
+    _config.loadBalancerExecutor = loadBalancerExecutor;
+    return this;
+  }
+
```

Comment:
It's unfortunate but you need to match the local code style here and use this formatting for braces:
```java
  public D2ClientBuilder setLoadBalancerExecutor(ExecutorService loadBalancerExecutor)
  {
    _config.loadBalancerExecutor = loadBalancerExecutor;
    return this;
  }
```

Reply:
Will keep with origin format. Thanks